### ISSUE TITLE
Issue: opens twice

### ISSUE THREAD

**Issue by cuddlebat** [archetyped/simple-lightbox#430]
## Description of Problem

The Pop Up opens twice on my page. See here: http://cenegenics-az.com/cenegenics-elite-performance-incinerate/

## Details

* URL of Page exhibiting problem: 
* Browser(s) exhibiting problem (Firefox, Chrome, etc.): 
* Version of SLB Installed: 
* Other plugins installed:

**Comment by cuddlebat** [comment]
It's the link that says "Enlarge Supplement Facts"

**Comment by archetyped** [comment]
Hi, I cannot find a link that says "Enlarge Supplement Facts" at the link you provided. Please provide a link to a page that exhibits the issue you are experiencing and I would be glad to take a closer look.

**Comment by cuddlebat** [comment]
Sorry, this will have to wait, I realized that the images I was going to lightbox are of too poor quality to be published and my boss wants the page live before I finish recreating the images to be lightboxed.

If you need to you can close this and I will repost it later. I just have to take care of the images first and it won't do to have the links on there for people to see the bad quality.

Thanks

PS I forgot I had posted this otherwise I would have deleted it. Sorry to have wasted your time.

**Comment by cuddlebat** [comment]
Okay, I am back. The issue is still happening. http://cenegenics-az.com/adrenal-support/

Look at the blue link that says "View Supplement Facts" under the black and white table image. Click it. It will pop up twice. I need it to pop up only once.

Thanks,

**Comment by archetyped** [comment]
Hi, it appears that you have two lightbox plugins running on the page.  Using multiple lightbox plugins is not recommended as they are designed to activate the same types of links (e.g. image links), which can lead to multiple lightboxes being displayed as you are experiencing.

Removing one of lightbox plugins is recommended to resolve your issue.
